Resources/ServerInfo/_NF/Guidebook/Shipyard/Brigand.xml

```diff
@@ -19,24 +19,7 @@
 	[color=#a4885c]Available Roles:[/color] Contractor, Pilot, Mercenary
 	
   "Civilian conversion of an old military light frigate from the early days of humanity's expansion to the stars. The original light frigate design was made in pre-FTL era of technology."
-  
-  # Piloting
-  <Box>
-    <GuideEntityEmbed Entity="ComputerShuttle"/>
-  </Box>
-  After clicking the shuttle console, you should see a radar view of the shuttle. Here are the steps for piloting the shuttle back and forth:
-
-  - First, disconnect any airlocks that are connected to the dock.
-  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with [color=#028ed9]"W"[/color] and [color=#028ed9]"S"[/color] being forward and backward, [color=#028ed9]"A"[/color] and [color=#028ed9]"D"[/color] being left and right, and [color=#028ed9]"Q"[/color] and [color=#028ed9]"E"[/color] being rotating left and right; and [color=#028ed9]"spacebar"[/color] being the brake and moving precicely by holding the [color=#028ed9]"spacebar"[/color] while doing other inputs.
-  
-  # Hiring crew
-  <Box>
-    <GuideEntityEmbed Entity="ComputerStationRecords"/>
-    <GuideEntityEmbed Entity="ClothingHeadsetGrey"/>
-    <GuideEntityEmbed Entity="IntercomCommon"/>
-  </Box>
-  As a Captain of a Nanotrasen vessel, you have the authority to hire, fire, demote, or promote crew members at will. There are two ways you as a Captain can go about hiring crew: you can either use the [color=#a4885c]station records computer[/color] on your ship to open crew slots (jobs available vary from ship to ship) or invite through in-game communications other players to your crew.
-  
+    
   # Salvage expeditions
```

Comment:
This would be another useful entry under Guidebook.

Reply:
Good point, I'll add it soon.